### PR TITLE
[Feature] Add Resources Section

### LINKED ISSUE #288: Home and information screen changes
There are some modifications that need to be made to the home and information screens to better surface information:

- [x] Add new information section to home screen
- [x] Add phone icon to home screen
- [x] Update information screen, removing items
- [x] Add "last updated date" to the information screen
- [x] Add contributors under the Acknowledgements and references

# Add new information section to home screen
A new "information" section needs to be added to the home screen, after the ICU section.  It should contain two new buttons

**WH Definitive resources**
This needs to lead to a subscreen which has the first 3 buttons from the current information page.  These are:

1. WH Novel coronavirus website
2. WH anaesthetic microsite
3. WH ICU microsite

**Additional resources**
This is the screen which the existing "Additional resource" button in the existing information screen leads to

# Add phone icon to home screen
On the home screen, we need to add a new phone icon to the right of the current "information" icon.  The phone icon should take you to the website which the existing "Contact numbers" button in the existing information screen leads to ( http://wh.cyphix.net/ )

# Update information screen
The first 5 buttons in the information screen can now be removed.  The only items remaining on the information screen should be:

- App feedback
- Disclaimer and conditions of use
- Acknowledgements and references
- Third party licenses
- WHAC19 Icon to remain present

# Add "last updated date" to the information screen
So that users can gauge how up to date the information in the app is we need to show the date their build was created.  Hopefully this should be prominently displayed close to the app icon, in the information screen.  Given the info screen has been re-ordered it might even be nice to give it a little more ✨  by moving the icon around or otherwise shuffling around things.

From a coding point of view, ideally this date should be based off the build time, so we don't have to update a string every time we do a release.

# Add contributors under the Acknowledgements and references
We want to add a new section to the Acknowledgements and references screen which lists all the contributors ❤️

### PR DESCRIPTION
**Pull request by yimingc2**
Resolved #288 

## Abstract
This PR will add a new resource section in the home screen, update information view and the home page.

- [x] `release-notes.txt` has been updated.
(👆 delete this line if your changes have no user facing features)

## Changes

- Add phone icon to home screen
- Add Resource Section to home screen
- Move "WH Novel Coronavirus Website", "WH Anaesthetic Microsite" and "WH ICU Microsite" from the info view to WH Definitive Resources
- Add contributors

## Screenshots
Simulator Screen Shot - iPhone 11 Pro - 2020-04-28 at 19 33 02
Simulator Screen Shot - iPhone 11 Pro - 2020-04-28 at 19 33 07
Simulator Screen Shot - iPhone 11 Pro - 2020-04-28 at 19 33 57
Simulator Screen Shot - iPhone 11 Pro - 2020-04-28 at 19 34 16

## Things to note
- "Last updated date" is not implemented in this PR. It needs more discussion and a separate PR.
- The title color of Additional Resources Screen is black instead of white. I don't know if we should create a new text style for the white color as it's only used in this page.

### CODE REVIEW

**Comment by workerbee22** [comment]
As discussed last night @yimingc2 we should probably separate **Acknowledgements & References** into two separate screens:
1. References ... just the references section
2. Acknowledgements ... which has the Acknowledgements and Contributors sections

**Comment by workerbee22** [comment]
@greggmiller Also suggest moving the Western Health **COVID-19 Daily Bulletin** out of **Additional Resources** ? It seems wrong in there, because it is not in the **WH Definitive Resources** section ... when it is in fact WH specific.

Also suggest renaming **WH Definitive Resources** to just **WH Resources** ? The word definitie seems redundant?????

Also suggest the words Microsite and Website not be capitalised????

**Review by workerbee22 — CHANGES_REQUESTED** [review]
Still need to split the Acknowledgements + Resources screen into 2 screens.

**Comment by greggmiller** [comment]
Agree with Mike’s comments. “Definitive” is redundant and other changes seem sensible.

**Comment by kimngyn** [comment]
I think we should keep the capitalisation for “microsite” and “website”. We have made it consistent for all titles in each cell within the app already. Please refer to other screens

File: lib/view/acknowledgements_view.dart [hunk 1]
```diff
@@ -67,6 +67,33 @@ Several companies who have provided support and employee time, including <a href
 <br/>
 <br/>
 Those who have given helpful and timely advice, including Byron Teu.<br/>
+  ''';
+  static const _contributorsHtml = '''
```
Comment (lukesleeman) [inline lib/view/acknowledgements_view.dart:34]: ❤️ Great to see this in here

File: lib/view/acknowledgements_view.dart [hunk 2]
```diff
@@ -67,6 +67,33 @@ Several companies who have provided support and employee time, including <a href
...
+<b>Adam Koch</b> - Flutter Dev<br/>
+<b>Brett Morgan</b> - Flutter Dev<br/>
+<b>Eric Jiang</b> - Flutter Dev<br/>
+<b>Dylan Lange</b> - Flutter Dev<br/>
+<b>Jonathan Galtieri</b> - Anaesthetist<br/>
+<b>Ben Blackman</b> - Anaesthetic Registrar<br/>
+<b>Fabien Dade</b> - ICU Registrar<br/>
+<b>Alice Gynther</b> - Anaesthetist<br/>
+<b>Kim Nguyen</b> - Flutter Dev<br/>
+<b>Bramley</b> - Flutter Dev<br/>
```
Comment (lukesleeman) [inline lib/view/acknowledgements_view.dart]: @rockgecko-development  - do you want us to add your second name?  What is it?!

File: lib/view/acknowledgements_view.dart [hunk 3]
```diff
@@ -67,6 +67,33 @@ Several companies who have provided support and employee time, including <a href
...
+<b>Suesi Tran</b> - Flutter Dev<br/>
+<b>June Chen</b> - Flutter Dev<br/>
+<b>Joshua Case</b> - Flutter Dev<br/>
+<b>Poornima Sivakumar</b> - Documentation<br/>
+<b>Fabio</b> - Documentation<br/>
```
Comment (lukesleeman) [inline lib/view/acknowledgements_view.dart:56]: @fabiendade   - do you want us to add your second name?  What is it?!

File: lib/view/resources/additional_resources_view.dart [hunk 4]
```diff
@@ -45,12 +45,12 @@ class AdditionalResourcesView extends StatelessWidget {
...
   Widget build(BuildContext context) {
     return Scaffold(
-        backgroundColor: AppColors.appBackground,
+        backgroundColor: AppColors.backgroundPurple,
         appBar: AppBar(
           // Warning brightness interacts with SystemUiOverlayStyle
           // See system_bars.dart comments
           brightness: Brightness.light,
-          backgroundColor: AppColors.appBarBackground,
+          backgroundColor: AppColors.purple50,
```
Comment (lukesleeman) [inline lib/view/resources/additional_resources_view.dart:53]: As part of the change, we need to alter the headings to be white:

| Designs in Zeplin  | Screenshot of current app |
| ------------- | ------------- |
| Resources : Additional Resources   | image  |

**Comment by yimingc2** [comment]
Hi team, based on the discussion above, I'm going to do these changes:
1. Separate Acknowledgements & References into two separate screens
2. Rename "WH Definitive Resources" to just "WH Resources"
3. Keep capitalisation for “Microsite” and “Website”

**Review by lukesleeman — COMMENTED** [review]
Nice work!  I found one small UI inconsistency.  Apart form that, once you have addressed the things @workerbee22 suggested we should be good to merge this PR 👍 🙌

File: lib/view/acknowledgements_view.dart (hunk 1 above)
Comment (yimingc2) [reply lib/view/acknowledgements_view.dart:34]: Built by ❤️

File: lib/view/acknowledgements_view.dart (hunk 2 above)
Comment (yimingc2) [reply lib/view/acknowledgements_view.dart]: Yeah, I just copy and paste from Zeplin. Would be good to let everyone check their names.

File: lib/view/resources/additional_resources_view.dart (hunk 4 above)
Comment (cjmlgrto) [reply lib/view/resources/additional_resources_view.dart:53]: +1 on that please! ☝️ 🙏  I know it's an exception to the styles, but it'd really help readability with these screens on top of that purple.

**Comment by yimingc2** [comment]
Nah it’s good to have it. Already updated. You should be able to see the
change now

File: lib/view/reference_view.dart [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+                child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.stretch,
+                    children: [
+                  Padding(
+                    padding: _titlePadding,
+                    child: const Text('References', style: Styles.textH4),
+                  ),
+                  ..._sections.map((e) =>
+                      ReusableCard.fromData(card: e, height: _sectionHeight))
+                ]))));
```
Comment (maks) [inline lib/view/reference_view.dart:74]: please add trailing commas for better formatting

File: lib/view/resources/definitive_view.dart [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+    ),
+    ReusableCard(
+      title: Strings.informationWhIcuTitle,
+      height: _cardHeight,
+      elevation: _cardElevation,
+      routeTo: whIcuMicrositeURL,
+    ),
+  ];
+
+  static Widget _buildSpacer({double height = 16}) => Container(height: height);
```
Comment (maks) [inline lib/view/resources/definitive_view.dart]: could we name `16` as const, seems to be used in more than one place in this file

**Review by maks — APPROVED** [review]
Great work! Just a couple of small formatting suggestions

**Review by workerbee22 — APPROVED** [review]

**Comment by greggmiller** [comment]
Looks great @yimingc2 🙌 💯  Thank you

**Comment by sallychung** [comment]
✅ v1.0.168

**Pull request merged**